src/app/jobs/job.component.ts

```diff
@@ -0,0 +1,54 @@
+// Copyright 2018 Novo Nordisk Foundation Center for Biosustainability, DTU.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {Component, OnInit} from '@angular/core';
+import {ActivatedRoute} from '@angular/router';
+
+import {JobService} from './jobs.service';
+import {Job} from './types';
+
+
+@Component({
+  selector: 'app-job',
+  templateUrl: './job.component.html',
+  styleUrls: ['./job.component.scss'],
+})
+export class JobComponent implements OnInit {
+  job: Job;
```

Comment:
Since we're dealing with data trough NgRx we don't really attach props to components, only their observables. This makes the markup usually more simple (no need for `*ngIf="job"`)